Fixing some spelling (#393)
chubin/cheat.sh: share/intro.txt

cheat.sh queries as well as search queries have many options.
They can be specified as a part of the query string in the URL, after {2?}.

> Short single letter options can be written all jointly or together.

Reviewer: ```suggestion
Short single letter options can be joined together.
```

Because "jointly" is the same and "together", meaning that -o -p -q can be -opq.

> Short single letter options can be written all jointly or together.
> Long options are separated with {2&}. For example, to switch 
> syntax highlighting off the {2T} switch is used:

Reviewer: ```suggestion
Short single letter options can be joined together. Long options are
separated with {2&}. For example, to switch syntax highlighting off
the {2T} switch is used:
```

    {1curl cht.sh/tar}{2?T}

A full list of all available cheat.sh options as well as description of all modes
of operation can be found in {2/:help}.

    {1curl cht.sh}{2/:help}

## cht.sh client

Though it's perfectly possible to access cheat.sh using {1curl} (or any other
HTTP client) alone, there is a special client that has several advantages
over plain curling: {2cht.sh}.

To install the client in {2~/bin}:

    {1curl} {2https://cht.sh/:cht.sh} {1> ~/bin/cht.sh}
    {1chmod +x ~/bin/cht.sh}

> Queries look the same, but instead of separate words in the query with {1+}
> like when using curl, you can use {1spaces} which looks more natural:

Reviewer: ```suggestion
Queries look the same, but you can use {1spaces} to separate words instead of {1+} used
by curl.
```
